src/test/java/com/puppycrawl/tools/checkstyle/checks/blocks/LeftCurlyCheckTest.java

```diff
@@ -307,31 +307,46 @@ public void testNewLineOptionWithLambda() throws Exception {
                 expected);
     }
 
+    @Test
+    public void testEolSwitch() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(LeftCurlyCheck.class);
+        checkConfig.addAttribute("option", LeftCurlyOption.EOL.toString());
```

Comment:
please add tests for all other modes - http://checkstyle.sourceforge.net/property_types.html#lcurly
for second nlow case ... try 
```
case 
       VALUE:
```

Reply:
done